### PR TITLE
Code review topic revised as topic was too long.

### PR DESCRIPTION
**Pull request by rmacken98**
Code review topic revised and changes made due to overlap in points made and topic being too long.
Files renamed :
CRBP.md -> CodeReviewBadPractices.md
image002.gif -> linesofcodegraph.gif

### CODE REVIEW

File: CodeReviews/CodeReviewBadPractices.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+When reviewing others code it is best to remember that it is not a competition to see who can code better. Software developers are often proud of their solutions and take criticism personally on occasion. Reviews help us see how others thought process occurs and feedback given should be constructive uplifting and insightful to help others program more efficiently. Be careful to always provide feedback that does not question the person but questions how and why they created a solution the way they did. However if the developer has done something right and you found how they've written their code or how they implemented a functionality to be good/clever let them know.
 
 <p align="center">
 <img src="https://miro.medium.com/max/1400/1*D9YKlFGZKiYoR57tUTpCrg.jpeg" alt="T-Shirt Size Image" width="400">
```
Comment (NicBoi) [inline CodeReviews/CodeReviewBadPractices.md]: Should reference the images in the repo and modify alternative text to a more appropriate name

File: CodeReviews/CodeReviewBadPractices.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 
 * **Meaningful comments**  
-If a person makes an error while programming it is almost certain that this error has been repeated across in other parts. When reviewing others code do not point out every occurrence of the error instead leave a detailed not explaining the overall issue. It is essential to reply to all comments in order to ensure that the review process is actually being completed.
+When reviewing others code do not point out every occurrence of an error instead leave a detailed note explaining the overall issue. 
+Ensure there is a balance between simply pointing out problems for them to fix and giving explicit instructions on how to fix. As giving explicit instructions can deny them the opportunity to learn from their mistakes.
+It is essential to reply to all comments in order to ensure that the review process is actually being completed. 
+
 
 <p align="center">
 <img src="https://miro.medium.com/max/1400/1*UbeT74oIf3JlX1YRu90EKA.jpeg" alt="T-Shirt Size Image" width="400" center>
```
Comment (NicBoi) [inline CodeReviews/CodeReviewBadPractices.md]: Should reference the images in the repo and modify alternative text to a more appropriate name

**Review by NicBoi — CHANGES_REQUESTED** [review]
I think you need to move the images from "Bad_Practices/Code_Reviews/Images" to your topic folder and update the image links

**Review by Lordjiggyx — APPROVED** [review]
Happy with this

**Review by NicBoi — APPROVED** [review]
Looks good, happy to merge

**Pull request merged**